tests/apollo/CMakeLists.txt

```diff
@@ -1,33 +1,39 @@
+if (BUILD_ROCKSDB_STORAGE)
+    set(TESTS_ARGS = "-p")
+else()
+    set(TESTS_ARGS = "")
+endif()
+
 add_test(NAME skvbc_basic_tests COMMAND sh -c
-        "python3 -m unittest test_skvbc 2>&1 > /dev/null"
+        "TESTS_ARGS=${TESTS_ARGS} python3 -m unittest test_skvbc 2>&1 > /dev/null"
```

Comment:
I think it would be more elegant to inject BUILD_ROCKSDB_STORAGE here (and all environment variables, ideally)

Reply:
Yes, a map of all environment variables would be better in the long run if more options appear, only the "start_replica_cmd" in the python code has to be modified.